presto-cli/src/main/java/io/prestosql/cli/StatusPrinter.java

```diff
@@ -208,9 +208,9 @@ public void printFinalInfo()
 
             // Spilled Data: 20GB, Writing Time: 22s, Reading Time: 1.2s
             if (stats.getSpilledBytes() > 0) {
-                Duration readTime = millis(stats.getElapsedSpillReadTimeMillis());
-                Duration writeTime = millis(stats.getElapsedSpillWriteTimeMillis());
-                String summary = String.format("Spilled: %s, Writing Time: %.1fs, Reading Time: %.1fs",
+                Duration readTime = millis(stats.getElapsedSpillReadTimeMillis() / nodes);
```

Comment:
what if all nodes don't flush? use the count of the nodes that did

Reply:
Fixed